tools-local/prebuilt-baseline-offline.xml

```diff
@@ -240,6 +240,7 @@
     <Usage Id="Microsoft.NETCore.Targets" Version="2.0.0" />
     <Usage Id="Microsoft.NETFramework.ReferenceAssemblies" Version="1.0.0-preview.2" IsDirectDependency="true" IsAutoReferenced="true" />
     <Usage Id="Microsoft.NETFramework.ReferenceAssemblies" Version="1.0.0" IsDirectDependency="true" IsAutoReferenced="true" />
+    <Usage Id="Microsoft.NETFramework.ReferenceAssemblies.net20" Version="1.0.0-preview.2" />
```

Comment:
Added this preemptively to https://github.com/dotnet/source-build/wiki/Driving-to-zero-prebuilt-dependencies-in-5.0.